eixampledb-api/src/test/java/com/eixampledb/api/MainControllerTest.java

```diff
@@ -151,7 +148,7 @@ public void shouldDoNothing() throws Exception {
                 .andExpect(content().string("I'm a word"));
 
         mockMvc.perform(put("/actualkey8").header("op", "INCR"))
-                .andExpect(status().isOk());
```

Comment:
This should not happen. The request is bad, it can not increment a String therefore it should throw a bad request status code to tell the user that the request is wrong. Otherwise the user may think that the value has been actually incremented, since it returned a 200.